components/script/dom/bindings/str.rs

```diff
@@ -28,10 +29,21 @@ impl ByteString {
         str::from_utf8(&vec).ok()
     }
 
+    /// Returns ownership of the underlying Vec<u8> the ByteString is
+    /// unusable afterwards as it has been moved
+    pub fn own_bytes(self) -> Vec<u8> {
```

Comment:
Ah! Good point! I thought about something like `drain_bytes`, but we'll go with `into_bytes`.